packages/server/prod-server/src/libs/render/index.ts

```diff
@@ -73,6 +78,20 @@ export const createRenderHandler: CreateRenderHandler = ({
         ctx.headers[`x-${cutNameByHyphen(metaName)}-ssr-fallback`]);
     if (route.isSSR && !useCSR) {
       try {
+        const userAgent = ctx.getReqHeader('User-Agent') as string | undefined;
+        if (shouldFlushServerHeader(conf.server, userAgent)) {
```

Comment:
Why isn't this feature written in a server plugin? I think maybe we should keep the prod-server as small as possible.